src/ioc/index.ts

```diff
@@ -1,12 +1,19 @@
 import IoCContainer from 'ioc-lite';
 
-// import { Logger } from '../services/logger';
-// import { HTTP } from '../services/http';
-// import { Users } from '../services/users';
+import { Logger } from '../services/logger';
+import { HTTP } from '../services/http';
+import { Users } from '../services/users';
+
+const config = (window as any).__CONFIG__;
+delete (window as any).__CONFIG__;
+
 
 export const createIoCContainer = () =>  {
   const ioc = new IoCContainer();
-  // you can register some resources right now below...
+  ioc.registerClass('logger', Logger);
+  ioc.registerClass('http', HTTP);
+  ioc.registerClass('users', Users);
 
+  ioc.register('apiConfig', {path: config.api.path, resources: config.api.resources});
```

Comment:
you can just pass `config.api` object, something like it : `{ ...config.api }`

Reply:
also, you can register `'apiConfig'` anywhere, so you can keep getting config from the window in the main (index.ts) file